packages/clay-select/src/index.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {SelectHTMLAttributes} from 'react';
+
+export interface ISelectOption {
+	label: string;
+	selected?: boolean;
+	value: string;
+}
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+	/**
+	 * Props to add to the outer most container.
+	 */
+	containerProps?: React.HTMLAttributes<HTMLDivElement>;
+
+	/**
+	 * Flag to indicate if the select is disabled or not.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Id attribute of the select.
+	 */
+	id?: string;
```

Comment:
Same as above, you probably don't need to specify this since it already extends from `SelectHTMLAttributes<HTMLSelectElement>` which should contain that.